pages/docs/security.mdx

```diff
@@ -0,0 +1,114 @@
+---
+description: Survey of common security problems facing LLM-based applications and how to use Langfuse to trace, prevent, and evaluate security risks.
+---
+
+# Security with Langfuse
+
+There are a host of potential security risks involved with LLM-based applications, such as prompt injection, personally identifiable information (PII), or harmful prompts. Langfuse provides a comprehensive and easy way to monitor and protect against these security risks.
```

Comment:
'leakage of' PII

Reply:
Langfuse provides a comprehensive and easy way to monitor and protect against these security risks.

to 

Langfuse can be used to monitor and protect against these security risks.